README.md

```diff
@@ -1,15 +1,30 @@
-# Ocean Protocol Subgraph
+[![banner](https://raw.githubusercontent.com/oceanprotocol/art/master/github/repo-banner%402x.png)](https://oceanprotocol.com)
 
-## Running locally
+<h1 align="center">ocean-subgraph</h1>
+
+> 🦀 Ocean Protocol Subgraph
+
+[![Build Status](https://travis-ci.com/oceanprotocol/ocean-subgraph.svg&branch=main)](https://travis-ci.com/oceanprotocol/ocean.js)
+[![js oceanprotocol](https://img.shields.io/badge/js-oceanprotocol-7b1173.svg)](https://github.com/oceanprotocol/eslint-config-oceanprotocol)
+
+- [🦑 Development](#-development)
+- [✨ Code Style](#-code-style)
+- [⬆️ Releases](#️-releases)
+- [🛳 Production](#-production)
+- [⬆️ Deployment](#️-deployment)
+- [🏛 License](#-license)
+
+## 🦑 Development
 
-* Install Graph CLI globally with npm
 ```bash
-npm install -g @graphprotocol/graph-cli
+npm i
+npm start
```

Comment:
this is the goal. Developers should only have to run `npm start` to get it running locally. Would be fine to tell devs to bring up local graph node first, but our instructions are crazy complicated for that